pkgs/development/compilers/llvm/4/default.nix

```diff
@@ -20,7 +20,7 @@ let
   # Add man output without introducing extra dependencies.
   overrideManOutput = drv:
     let drv-manpages = drv.override { enableManpages = true; }; in
-    drv // { man = drv-manpages.man; /*outputs = drv.outputs ++ ["man"];*/ };
+    drv // { man = drv-manpages.out; /*outputs = drv.outputs ++ ["man"];*/ };
```

Comment:
Is this still necessary?

Reply:
It's less painful, as the pages are relatively cheap to build.  I suppose it can be useful for uniformity (e.g. `programs.man.enable` in `configuration.nix`).

Reply:
I believe so, unfortunately :P.  While this makes the manpages much cheaper to build and have a tiny runtime closure, the build-time dependencies are still numerous but most problematically is that they include things like Sphinx (python package) that shouldn't force Darwin rebuilds.

Reply:
BTW, the build-time closure of man pages _could_ be cut down, somewhat.